src/types.ts

```diff
@@ -1,22 +1,39 @@
-import { PstState } from 'warp-contracts';
+/**
+ * Copyright (C) 2022-2024 Permanent Data Solutions, Inc. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
 
-export type ANTState = PstState & {
+export type ANTState = {
   ticker: string; // A short token symbol, shown in block explorers and marketplaces
-  name: string;   // The friendly name of the token, shown in block explorers and marketplaces
-  owner: string;  // The owner of this contract who can execute specific methods
+  name: string; // The friendly name of the token, shown in block explorers and marketplaces
+  owner: string; // The owner of this contract who can execute specific methods
   controllers: string[]; // The controller of the records, who can add/change subdomains and their settings
-  records: {     // A list of all subdomains and their corresponding Arweave Transaction IDs and TTLs
-    [subDomain: string]: { // the subdomain used, default is the root @
-      transactionId: string, // The transaction ID that the subdomain points to.
-      ttlSeconds: number // The amount of time (in seconds) this transaction is cached for, default at 900 seconds
-    }
+  records: {
+    // A list of all subdomains and their corresponding Arweave Transaction IDs and TTLs
+    [subDomain: string]: {
+      // the subdomain used, default is the root @
+      transactionId: string; // The transaction ID that the subdomain points to.
+      ttlSeconds: number; // The amount of time (in seconds) this transaction is cached for, default at 900 seconds
+    };
```

Comment:
create types:

```typescript
export type ArNSRecordData = {
   transactionId: string,
   ttlSeconds: number,
}

export type ArNSRecords = Record<string, ArNSRecordData>;
```

and set the type here
```
    records: ArNSRecords
```

Reply:
added